lib/shoryuken/queue.rb

```diff
@@ -58,33 +68,48 @@ def sanitize_messages!(options)
       options
     end
 
+    def add_fifo_attributes!(options)
+      return unless fifo?
+
+      options[:message_group_id]         ||= MESSAGE_GROUP_ID
+      options[:message_deduplication_id] ||= Digest::SHA256.digest(options[:message_body])
```

Comment:
@richseviora so in here I think we can set `message_group_id` and `message_deduplication_id` in case they are not supplied. I want to avoid checking `has_content_deduplication?`, otherwise we would need do a request to check if it's set or not per message, like N+1. WDYT?